Source/WebKit/NetworkProcess/WebStorage/StorageManager.cpp

```diff
@@ -231,8 +229,8 @@ LocalStorageNamespace* StorageManager::getOrCreateLocalStorageNamespace(StorageN
     if (!m_localStorageNamespaces.isValidKey(storageNamespaceID))
         return nullptr;
 
-    return m_localStorageNamespaces.ensure(storageNamespaceID, [this, storageNamespaceID]() {
-        return makeUnique<LocalStorageNamespace>(*this, storageNamespaceID);
+    return m_localStorageNamespaces.ensure(storageNamespaceID, [this, storageNamespaceID, quota = m_quotaInBytes]() {
```

Comment:
As you're capturing this, you should be able to access m_quotaInBytes directly inside the lambda.

Reply:
Yeah, will skip capturing m_quotaInBytes member separately